solvebio/resource/object.py

```diff
@@ -0,0 +1,69 @@
+"""Solvebio Object API resource"""
+from ..help import open_help
+from ..client import client
+
+from .solveobject import convert_to_solve_object
+from .apiresource import CreateableAPIResource
+from .apiresource import ListableAPIResource
+from .apiresource import SearchableAPIResource
+from .apiresource import UpdateableAPIResource
+from .apiresource import DeletableAPIResource
+
+
+class Object(CreateableAPIResource,
```

Comment:
Does it make sense to add helper methods to Objects, such as children/parent/siblings/etc... to make it easier to traverse vaults?